Sources/Logging/Logging.swift

```diff
@@ -62,7 +62,7 @@ public struct Logger {
     }
 
     @inlinable
-    public subscript(metadataKey metadataKey: String) -> String? {
+    public subscript(metadataKey metadataKey: String) -> Logging.MetadataValue? {
```

Comment:
Nitpick, though would be nice to be consistent: s/`Logging.MetadataValue`/`Logging.Metadata.Value`/g

Since we write `Logging.Metadata.Key` and that's our only way of writing the key nicely, I'd suggest to sticking with `Logging.Metadata.Value` (as we do already in some places, e.g. `Logging.Metadata.Key) -> Logging.Metadata.Value?`)

Reply:
yes, that's cool! You can now use that spelling but we do need a type called `MetadataValue` or smth because we can't create a type nested in the typealias. But because the typealias is to a dictionary, we can use `Metadata.Value` (and .Key) now

Reply:
Yeah, the type remains ofc :) The spelling when we refer to it in signatures would be nice as `.Value` since symmetry with `.Key`, sounds good? :)